packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -127,38 +127,41 @@ export class BlockSwitcher extends Component {
 				} }
 				renderContent={ ( { onClose } ) => (
 					<>
-						{ hasBlockStyles &&
-							<PanelBody
-								title={ __( 'Block Styles' ) }
-								initialOpen
-							>
-								<BlockStyles
-									clientId={ blocks[ 0 ].clientId }
-									onSwitch={ onClose }
-									onHoverClassName={ this.onHoverClassName }
-								/>
-							</PanelBody>
+						{ ( hasBlockStyles || possibleBlockTransformations.length !== 0 ) &&
+							<div className="components-popover__container">
```

Comment:
Thiis className doesn't follow our guidelines. It should be something like `block-editor-block-switcher__container` or something like that. https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#naming

Reply:
since I was trying to keep compatibility with `components-popover__content` i went with that name, but it makes more sense to have it `block-editor-block-switcher__container` since it's inside that file, will change.